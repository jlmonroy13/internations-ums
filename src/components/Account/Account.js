import React from 'react';
import PropTypes from 'prop-types';

function Account({ children }) {
  return (
    <div className="account">
      <div className="grid grid--center">{children}</div>
    </div>
  );
}

Account.propTypes = {
  children: PropTypes.node.isRequired,
  title: PropTypes.string,
};

export default Account;
