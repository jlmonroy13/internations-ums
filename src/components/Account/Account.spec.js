import React from 'react';
import renderer from 'react-test-renderer';
import Account from './Account';

describe('<Account />', () => {
  it('should render the account component', () => {
    const wrapper = renderer.create(
      <Account>
        <span />
      </Account>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
