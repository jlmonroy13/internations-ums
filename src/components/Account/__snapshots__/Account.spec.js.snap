// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Account /> should render the account component 1`] = `
<div
  className="account"
>
  <div
    className="grid grid--center"
  >
    <span />
  </div>
</div>
`;
